src/05_lists.py

```diff
@@ -7,23 +7,30 @@
 # For the following, DO NOT USE AN ASSIGNMENT (=).
 
 # Change x so that it is [1, 2, 3, 4]
-# YOUR CODE HERE 
+# YOUR CODE HERE
+x.append(4)
 print(x)
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
-# YOUR CODE HERE 
+# YOUR CODE HERE
+x.extend([8, 9, 10])
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 10]
-# YOUR CODE HERE 
+# YOUR CODE HERE
+x.pop(4)
```

Comment:
Python has a remove method for lists which removes the object from the list. This would be handy if you didn't know exactly where the element was in the list.

x.remove(8)